ReactWindows/ReactNative.Shared/Views/TextInput/ReactPasswordBoxShadowNode.cs

```diff
@@ -260,7 +271,7 @@ private static string GetDefaultPasswordChar()
             if (s_passwordChar == null)
             {
                 var passwordBox = new PasswordBox();
-                s_passwordChar = passwordBox.PasswordChar;
+                s_passwordChar = passwordBox.PasswordChar.ToString();
```

Comment:
>ToString [](start = 58, length = 8)

Technically this is not needed for UWP.  It's a wasted method call. Serious nitpick, but in general I don't know where I stand on sharing implementations for view managers at all.  I really think we should have an all or nothing approach for view managers, I think we'll want to evolve the view managers for WPF separately from UWP.